blazor/dropdown-list/localization.md

```diff
@@ -7,6 +7,23 @@ control: DropDown List
 documentation: ug
 ---
 
-# Localization in Blazor DropDown List Component
+# Globalization and Localization in Dropdown List
+
+## Localization
+
+[Blazor DropDownList](https://www.syncfusion.com/blazor-components/blazor-dropdown-list) component can be localized. Refer to [Blazor Localization](https://blazor.syncfusion.com/documentation/common/localization) topic to localize Syncfusion Blazor components.
+
+## Globalization
+
+### Enable RTL mode
+
+You can specifies [EnableRtl](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.DropDownListModel-2.html#Syncfusion_Blazor_DropDowns_DropDownListModel_2_EnableRtl) as boolean value that indictes to Enable or disable rendering component in right to left direction. For writing systems will require Arabic, Hebrew, and more. The direction can be switched to right-to-left.
```

Comment:
Specify the [EnableRtl](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.DropDownListModel-2.html#Syncfusion_Blazor_DropDowns_DropDownListModel_2_EnableRtl) as a boolean value that indicates to enable or disable rendering component in the right to left direction. Writing systems will require Arabic, Hebrew, and more. The direction can be switched to right-to-left.